### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by dholms**
## Problem
Travis is always failing

## Solution
Fix linting errors, remove code coverage reqs (until we have more tests)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -9,4 +9,17 @@ module.exports = {
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
   ],
+  rules: {
```
Comment (dholms) [inline .eslintrc.js:12]: Let me know your thoughts on these rules!
`@typescript-eslint/no-use-before-define` was causing it to error when using a function before it's defined. In Haskell, we normally put the most important function at the top of a file, and helper functions beneath it.

`'@typescript-eslint/member-delimiter-style'` was set to a semi-colon by default so for instance, types had to be laid out like:
```ts
type Fake = {
  key1: string;
  key2: string;
  key2: string;
}
```
It's mostly personal preference, but I much prefer the clean look of no delimiter. Thoughts?

Comment (icidasset) [reply .eslintrc.js:12]: Yeah same, only semicolons when absolutely necessary 😄 
Rules seem fine 👍

File: src/ffs/types.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export type Links = { [name: string]: Link }
 // ----
 
 export type NonEmptyPath = [string, ...string[]]
-export type SyncHook = (cid: CID) => any
+export type SyncHook = (cid: CID) => unknown
```
Comment (icidasset) [inline src/ffs/types.ts:44]: I did not know `unknown` was a thing 🤷

Comment (dholms) [reply src/ffs/types.ts:44]: yeah just recently found this out also! my understanding is that it's like `any` but doesn't completely blow out the type system

**Pull request merged**